packages/react-examples/src/react-badge/PresenceBadge/PresenceBadge.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { PresenceBadge } from '@fluentui/react-badge';
+import { SkypeMinusIcon, SkypeClockIcon, SkypeCheckIcon, CancelIcon, SkypeArrowIcon } from '@fluentui/react-icons-mdl2';
+
+export const PresenceBadgeStatus = () => (
+  <>
+    <h2>Circular (default)</h2>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        width: 200,
+      }}
+    >
+      <PresenceBadge status="busy" icon={<SkypeMinusIcon />} />
+      <PresenceBadge status="away" icon={<SkypeClockIcon />} />
+      <PresenceBadge status="available" icon={<SkypeCheckIcon />} />
+      <PresenceBadge status="oof" icon={<SkypeArrowIcon />} />
+      <PresenceBadge status="offline" icon={<CancelIcon />} />
```

Comment:
The icons should be set by `usePresenceBadge` based on the status; the user shouldn't need to specify them manually.

Reply:
But I am guessing we don't have converged icons yet, right?

Reply:
I made a similar decision for `react-menu`... we have no idea what the converged icon story is or will be. Additionally consumers will have different icon needs (i.e. Teams will need different icons than Office) initially.

Reply:
Yeah I mostly wanted to be sure that the plan was to include the icons once the new icons are available, and not require users to supply their own icons. In the meantime to avoid taking a dependency on mdl2 icons, you could either leave out the icons (and log an issue to add them later), or copy the mdl2 icon temporarily into the react-badge package as a placeholder. (See [DefaultAvatarIcon.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-avatar/src/components/Avatar/DefaultAvatarIcon.tsx) for an example)

Reply:
I still think user should be able to set the icons and not be internally in the `Badge` WDYT @ling1726 ?

Reply:
IMO if internal defaults make sense we should include them, quoting myself:
> If the suggestion is to copy icons into the package temporarily and use them as defaults, that's fine with me

To be clear the slots still need to exist for customization, if it makes sense to have default icons according to design, then we can copy SVGs internally for now until we know what the converged icon package will look like

Reply:
Added the default icons